option/User/setup.sh

```diff
@@ -14,6 +14,19 @@ pw_create_account ( ) {
     $CHOWN $UGID ${BOARD_FREEBSD_MOUNTPOINT}${HOME_DIR}/$1
 }
 
+pw_add_user_to_group ( ) {
+    echo "Adding user $1 to group $2"
+    $PW -V ${BOARD_FREEBSD_MOUNTPOINT}/etc/ groupmod $2 -m $1
+}
+
 # Add the specified account.
-strategy_add $PHASE_FREEBSD_BOARD_INSTALL pw_create_account $1
+# Ensure it happens before adding user to groups
+PRIORITY=110 strategy_add $PHASE_FREEBSD_BOARD_INSTALL pw_create_account $1
```

Comment:
You don't need to set PRIORITY here:  By default, strategies with the same priority are executed in the order they are added.  Since you're adding `pw_create_account` before `pw_add_user_to_group`, you get the right order for free.